src/users/users.controller.ts

```diff
@@ -16,5 +18,16 @@ export class UsersController {
     return this.userService.join(joinData);
   }
 
-  //TODO: POST login
+  @Post('login')
+  async login(@Res() response: Response, @Body() loginData: LoginDto): Promise<any> {
+    const jwt = await this.userService.login(loginData);
+
+    response.cookie('accessToken', jwt.accessToken, {
```

Comment:
요렇게는 어떨까요?
그리고 userService의 login method에서 return값이 하나임에도 불구하고 객체로 반환해주신 이유는 이후에 값이 추가될 수 있어서 그런걸까요?
```suggestion
    const { accessToken } = await this.userService.login(loginData);

    response.cookie('accessToken', accessToken, {
```

Reply:
토큰, 사용자 정보 같은 값을 보낼 수도 있을 것 같습니다.